src/rpdk/core/jsonutils/utils.py

```diff
@@ -103,6 +103,97 @@ def traverse(document, path_parts):
     return document, tuple(path), parent
 
 
+# pylint: disable=R0913,C0301
+# flake8: noqa=C901,B950
```

Comment:
* ```C901 is too complex``` In this case, I need to create a recursive loop to get all the list properties. I don't know of a easier way of solving this. 
* ```B950 line too long (121 > 80 characters)``` In this case, the line being referred to is part of the tests being run for this method.
* ```R0913 too many arguments``` Switching to recursive method in place of the iterative approach required the addition of the parameters.
* ```C0301  Line too long``` In this case, the line being referred to is part of the tests being run for this method.

Reply:
I think I agree with `C901 is too complex` and `R0913 too many arguments`